From b8e2633ace0f3d9d57e68c27db8f594b8a5ddd7e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Fri, 30 Aug 2024 01:48:29 +0200
Subject: [PATCH] Makefiles: add back l10n_compile targets

These are needed to run 'make l10n_compile' from the root directory.
---
 openpaperwork-core/Makefile | 6 ++++++
 openpaperwork-gtk/Makefile  | 6 ++++++
 paperwork-backend/Makefile  | 4 ++++
 paperwork-gtk/Makefile      | 4 ++++
 paperwork-shell/Makefile    | 4 ++++
 5 files changed, 24 insertions(+)

diff --git a/openpaperwork-core/Makefile b/openpaperwork-core/Makefile
index 87dca37fa..c7dd3fbf1 100644
--- a/openpaperwork-core/Makefile
+++ b/openpaperwork-core/Makefile
@@ -90,6 +90,12 @@ uninstall_c:
 l10n_extract:
 	$(CURDIR)/../tools/l10n_extract.sh "$(CURDIR)/src" "$(CURDIR)/l10n"
 
+l10n_compile:
+	$(CURDIR)/../tools/l10n_compile.sh \
+		"$(CURDIR)/l10n" \
+		"$(CURDIR)/src/openpaperwork_core/l10n" \
+		"openpaperwork_core"
+
 help:
 	@echo "make build || make build_py"
 	@echo "make check"
diff --git a/openpaperwork-gtk/Makefile b/openpaperwork-gtk/Makefile
index 961feb9b1..a706ad809 100644
--- a/openpaperwork-gtk/Makefile
+++ b/openpaperwork-gtk/Makefile
@@ -92,6 +92,12 @@ uninstall_c:
 l10n_extract:
 	$(CURDIR)/../tools/l10n_extract.sh "$(CURDIR)/src" "$(CURDIR)/l10n"
 
+l10n_compile:
+	$(CURDIR)/../tools/l10n_compile.sh \
+		"$(CURDIR)/l10n" \
+		"$(CURDIR)/src/openpaperwork_gtk/l10n" \
+		"openpaperwork_gtk"
+
 help:
 	@echo "make build || make build_py"
 	@echo "make check"
diff --git a/paperwork-backend/Makefile b/paperwork-backend/Makefile
index eac3bd0c9..98ff0d899 100644
--- a/paperwork-backend/Makefile
+++ b/paperwork-backend/Makefile
@@ -91,6 +91,10 @@ l10n_extract:
 	$(CURDIR)/../tools/l10n_extract.sh "$(CURDIR)/src" "$(CURDIR)/l10n"
 
 l10n_compile:
+	$(CURDIR)/../tools/l10n_compile.sh \
+		"$(CURDIR)/l10n" \
+		"$(CURDIR)/src/paperwork_backend/l10n" \
+		"paperwork_backend"
 
 help:
 	@echo "make build || make build_py"
diff --git a/paperwork-gtk/Makefile b/paperwork-gtk/Makefile
index 284b6e86b..436bdef03 100644
--- a/paperwork-gtk/Makefile
+++ b/paperwork-gtk/Makefile
@@ -117,6 +117,10 @@ l10n_extract:
 	$(MAKE) -C src/paperwork_gtk/model/help l10n_extract
 
 l10n_compile:
+	$(CURDIR)/../tools/l10n_compile.sh \
+		"$(CURDIR)/l10n" \
+		"$(CURDIR)/src/paperwork_gtk/l10n" \
+		"paperwork_gtk"
 
 help:
 	@echo "make build || make build_py"
diff --git a/paperwork-shell/Makefile b/paperwork-shell/Makefile
index 8fdae1f4e..0d550bcf2 100644
--- a/paperwork-shell/Makefile
+++ b/paperwork-shell/Makefile
@@ -90,6 +90,10 @@ l10n_extract:
 	$(CURDIR)/../tools/l10n_extract.sh "$(CURDIR)/src" "$(CURDIR)/l10n"
 
 l10n_compile:
+	$(CURDIR)/../tools/l10n_compile.sh \
+		"$(CURDIR)/l10n" \
+		"$(CURDIR)/src/paperwork_shell/l10n" \
+		"paperwork_shell"
 
 help:
 	@echo "make build || make build_py"
-- 
GitLab

